### DISCUSSION TITLE
comparison with vector database that supports sparse vector like milvus

### DISCUSSION THREAD

**Discussion by haowu2651** [xhluca/bm25s#66] [Q&A] [answered]
I'm using milvus with my database that is more than 4 m rows short text with both sparse vector (BM25) and dense vector (semantic embeddings), I don't know how is the comparison with it? 
If I use bm25s to fit the corpus, how to encode the my whole database and get sparse vector easily? would be faster with our lib?

**Comment by xhluca** [comment, marked as answer]
Feel free to take a look at this: https://github.com/xhluca/bm25s/blob/main/examples/index_and_retrieve_with_numba.py

Since it's database specific, you will ened to add code to fetch rows from your db. feel free to contribute a new example if you think the use case could help other users!
